app/src/main/java/org/oppia/app/help/faq/FAQViewModel.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.help.faq
+
+import org.oppia.app.viewmodel.ObservableViewModel
+
+/** [ObservableViewModel] for the recycler view of FAQActivity. */
+class FAQViewModel(val title: String) : ObservableViewModel()
```

Comment:
Instead of `ObservableViewModel ` make it `ViewModel ` only because the string won't change.

Reply:
Update comment also.

Reply:
done